### ISSUE TITLE
[BUG] Can not stop it under Windows

### ISSUE THREAD

**Issue by hantsy** [code-ready/crc#2738]
### General information

  * OS:  Windows
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it (Yes/No)? yes
  * Running CRC on: Laptop 

## CRC version
```bash
# Put `crc version` output here
❯ crc version
CodeReady Containers version: 1.32.1+4d40571f
OpenShift version: 4.8.5 (bundle installed at C:\Program Files\CodeReady Containers\crc_hyperv_4.8.5.crcbundle)
```
  
## CRC status
```bash
# Put `crc status --log-level debug` output here
❯ crc status
CRC VM:          Running
OpenShift:       Unreachable (v4.8.5)
Disk Usage:      0B of 0B (Inside the CRC VM)
Cache Usage:     14.13GB
Cache Directory: C:\Users\hantsy\.crc\cache
```

## CRC config
```bash
# Put `crc config view` output here
crc config view
- autostart-tray                        : false
- consent-telemetry                     : no
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows

❯ systeminfo

Host Name:                 HANTSY-T540P
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.19043 N/A Build 19043
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          hantsy
Registered Organization:
Product ID:                00330-50000-00000-AAOEM
Original Install Date:     8/20/2021, 1:17:33 PM
System Boot Time:          9/17/2021, 1:14:34 PM
System Manufacturer:       LENOVO
System Model:              20BFS38A00
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 60 Stepping 3 GenuineIntel ~2601 Mhz
BIOS Version:              LENOVO GMET86WW (2.34 ), 11/8/2018
Windows Directory:         C:\Windows
System Directory:          C:\Windows\system32
Boot Device:               \Device\HarddiskVolume1
System Locale:             en-us;English (United States)
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC+08:00) Beijing, Chongqing, Hong Kong, Urumqi
Total Physical Memory:     16,069 MB
Available Physical Memory: 1,836 MB
Virtual Memory: Max Size:  20,137 MB
Virtual Memory: Available: 3,008 MB
Virtual Memory: In Use:    17,129 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\HANTSY-T540P
Hotfix(s):                 4 Hotfix(s) Installed.
                           [01]: KB5004331
                           [02]: KB5000736
                           [03]: KB5005565
                           [04]: KB5005699
Network Card(s):           4 NIC(s) Installed.
                           [01]: Intel(R) Ethernet Connection I217-LM
                                 Connection Name: Ethernet
                                 Status:          Media disconnected
                           [02]: Intel(R) Dual Band Wireless-AC 7260
                                 Connection Name: Wi-Fi
                                 DHCP Enabled:    Yes
                                 DHCP Server:     172.30.12.1
                                 IP address(es)
                                 [01]: 172.30.12.50
                                 [02]: fe80::6d82:798a:1175:2c2e
                           [03]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (Default Switch)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 172.21.112.1
                                 [02]: fe80::69d6:db8e:1e33:dd5a
                           [04]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (WSL)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 172.22.128.1
                                 [02]: fe80::6993:f52:aeb6:5058
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.
```

### Steps to reproduce

```
❯ crc status
CRC VM:          Running
OpenShift:       Unreachable (v4.8.5)
Disk Usage:      0B of 0B (Inside the CRC VM)
Cache Usage:     14.13GB
Cache Directory: C:\Users\hantsy\.crc\cache
 hantsy@hantsy-t540p  D:  hantsylabs  spring-graphql-sample  java17 
❯ crc stop
INFO Stopping kubelet and all containers...
Executing systemctl action failed:  ssh command error:
command : sudo systemctl stop kubelet
err     : dial tcp 127.0.0.1:2222: connectex: No connection could be made because the target machine actively refused it.\n:
```

### Expected

Stop crc. It always started when system is started, can not stop it in the context menu of the icon tray or by command line. 

### Actual

None of them are working as expected.

**Comment by adrianriobo** [comment]
Duplicate of #2678

**Comment by adrianriobo** [comment]
Can we close this one in favor of #2678 as it is the same scenario?

**Comment by praveenkumar** [comment]
Closing in favor of #2678 

/close
